app/services/albums.js

```diff
@@ -0,0 +1,20 @@
+const request_promise = require('request-promise');
+const { logger } = require('express-wolox-logger');
+const api_url = require('../../config').common.albumsApiURL;
+
+const options = {
+  uri: `${api_url}`,
+  headers: {
+    'User-Agent': 'Request-Promise'
+  },
+  json: true
+};
+
+exports.getAlbums = id => {
```

Comment:
the method isn't only for get albums, also allow get album's photos

Reply:
I changed the method to return all albums or a specific album if you send an id, also added a separated method for albums photos

Reply:
but i think that it could be a single method, the only difference is a uri, what do you think?